src/app/routes/accountRecoveryRoute.js

```diff
@@ -73,14 +73,18 @@ module.exports = (app) => {
           const data = JSON.parse(proxyResData.toString('utf8'));
           if (data.responseCode === 'OK') {
             req.session.otpVerifiedFor = req.body;
-            const encrypt = {
+            const _encrypt = {
               key: req.body.request.key
             }
             if (req.body.request.userId) {
-              encrypt['id'] = req.body.request.userId
+              _encrypt['id'] = req.body.request.userId
             }
             var timeInMin = 5;
-            var validator = encriptWithTime(encrypt, timeInMin);
+            var validator = encriptWithTime(_encrypt, timeInMin);
+            const reqType = req.body.request.type;
+            const dataToEncrypt = {};
+            dataToEncrypt[reqType] = req.body.request.key;
+            req.session.otpEncryptedInfo = encrypt(JSON.stringify(dataToEncrypt));
```

Comment:
is two times encryption is required ?
can we include if condition like for user id ?

Reply:
`if` condition on what ?
type is always available in verify OTP route.